akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/ClusterShardingSettings.scala

```diff
@@ -310,7 +310,7 @@ final class ClusterShardingSettings(
   def withStateStoreMode(stateStoreMode: ClusterShardingSettings.StateStoreMode): ClusterShardingSettings =
     copy(stateStoreMode = stateStoreMode)
 
-  def withPassivateIdleEntitiesAfter(duration: FiniteDuration): ClusterShardingSettings =
+  def withPassivateIdleEntityAfter(duration: FiniteDuration): ClusterShardingSettings =
```

Comment:
The property that we change with this method is called `passivateIdleEntityAfter` and the Java API variant is using singular as well.